<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>-10</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="HumanList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>121</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Herp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Derp</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="ElfList">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Super</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Herp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Derp</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="DwarfList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>256</width>
       <height>131</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Cleric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mage</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dwarf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Elf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barbarian</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll AS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
